### DISCUSSION TITLE
[Feature] In-Album Indicator

### DISCUSSION THREAD

**Discussion by nothing2obvi** [immich-app/immich#10778] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In organizing assets in my timeline, it would be really helpful if there was an indicator that says whether or not the asset is already organized into an album (similar to the archived indicator when searching for photos). 

It would be even greater if you could hover over that indicator and Immich shows you which albums it is a part of. 

But at minimum, just an indicator showing whether the asset is already organized into an album would be great. It would help so much with organization and not wasting time organizing assets from your timeline into albums when they've already been added to albums.

### Platform

- [X] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Does the "not in album" search filter work for you?

![image](https://github.com/immich-app/immich/assets/11602424/97c0846f-0b57-45a1-9991-b9b971b9db26)

https://my.immich.app/search?query={"isNotInAlbum"%3Atrue}

**Comment by nothing2obvi** [comment]
No, but it's close. Those photos aren't laid out nicely as in the timeline, and you can't multi-select by clicking on one asset, holding shift, then clicking on the last asset of the set. You have to do it one-by-one.

Plus, seeing an in-album indicator on assets in the timeline would give peace of mind as you scroll through your timeline. You'd know that every asset is accounted for (i.e. placed in an album).

**Comment by bo0tzz** [comment]
#1621

**Reply by nothing2obvi** [reply]
Sounds good. I'll continue the conversation there.
